File: 1. Begginer/11- Pida al usuario dos números enteros (que se guardarán en las variables num1 y num2).Si num2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resto de l/main.go
package main
import "fmt"

func main() {
	var (
		num1 int
		num2 int
	)
	fmt.Println("Ingrese el primer número entero: ")
	fmt.Scanf("%d", &num1)
	fmt.Println("Ingrese el otro número entero: ")
	fmt.Scanf("%d", &num2)

	if (num2 == 0) {
		fmt.Println("Error: No se puede dividir un número entre 0")
	} else {
		fmt.Println("El resultado de dividir ", num1, " entre ", num2, " es igual a: ", num1/num2)		
	}
}

File: 1. Begginer/11- Pida al usuario dos números enteros (que se guardarán en las variables num1 y num2).Si num2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resto de l/main.cpp
#include <iostream>

using namespace std;

int main() {
	int num1;
	int num2;
	cout << "Ingrese el primer número entero: ";
	cin >> num1;
	cout << "Ingrese el segundo número entero: ";
	cin >> num2;
	if (num2 == 0) {
		cout << "Error: No se puede dividir un número entre 0";
	} else {
		cout << "El resultado de dividir " << num1 << " entre " << num2 << " es igual a: " << num1/num2;
	}
}